dcos/dcos-cluster-one-master.json

```diff
@@ -248,7 +248,7 @@
       "type": "string"
     }, 
     "linuxVersion": {
-      "defaultValue": "16.04.201605161", 
+      "defaultValue": "latest", 
```

Comment:
The current version (`16.04.201605161`) is missing so the template doesn't currently work.  I think we need a better story for ensuring that the templates in github work.  How do we choose/validate what image to use currently?  We need some sort of automated testing here.